### ISSUE TITLE
Jinja error during experiment creation on WCOSS

### ISSUE THREAD

**Issue by WalterKolczynski-NOAA** [NOAA-EMC/global-workflow#3087]
### What is wrong?

Following the early November upgrade to Cactus, experiment creation now fails due to a non-existent jinja attribute:
```
The create_experiment command (./create_experiment.py -y ../ci/cases/pr/C48_ATM.yaml --overwrite) failed with a non-zero status.  Output:
Traceback (most recent call last):
  File "./create_experiment.py", line 33, in <module>
    from wxflow import AttrDict, parse_j2yaml, Logger, logit
  File "/lfs/h2/emc/ptmp/walter.kolczynski/PR/PR_3053/global-workflow/workflow/wxflow/__init__.py", line 13, in <module>
    from .jinja import Jinja
  File "/lfs/h2/emc/ptmp/walter.kolczynski/PR/PR_3053/global-workflow/workflow/wxflow/jinja.py", line 16, in <module>
    @jinja2.pass_eval_context
AttributeError: module 'jinja2' has no attribute 'pass_eval_context'
```

### What should have happened?

`setup_expt.py` should create the experiment direction correctly.

### What machines are impacted?

WCOSS2

### What global-workflow hash are you using?

3FB68397

### Steps to reproduce

1. Clone on Cactus
2. Load the gw_setup module
3. Attempt to create an experiment

### Additional information

Presumably this will temporarily go away when they switch production, the reappear after Dogwood is upgraded the week of Nov 18.

See also NOAA-EMC/global-workflow#3086

**Comment by DavidHuber-NOAA** [comment]
Is there an issue open with the help desk?

**Comment by aerorahul** [comment]
The fix for this is simple.  Load the python module in `module_gwsetup.wcoss2.lua`.  For some reason, wcoss2 was using system python and not sure why that worked.

**Comment by JongilHan66** [comment]
> The fix for this is simple. Load the python module in `module_gwsetup.wcoss2.lua`. For some reason, wcoss2 was using system python and not sure why that worked.

@aerorahul Could you let me know how to load python module in  `module_gwsetup.wcoss2.lua` specifically?

**Comment by aerorahul** [comment]
> > The fix for this is simple. Load the python module in `module_gwsetup.wcoss2.lua`. For some reason, wcoss2 was using system python and not sure why that worked.
> 
> @aerorahul Could you let me know how to load python module in `module_gwsetup.wcoss2.lua` specifically?

@JongilHan66 
A fix was merged in #3053.  The change can be seen [here](https://github.com/NOAA-EMC/global-workflow/pull/3053/files#diff-4a96a6a951830b66c158250c0b7fca2be8240ed1b3e25059f9742cd419f91073).

**Comment by JongilHan66** [comment]
> > > The fix for this is simple. Load the python module in `module_gwsetup.wcoss2.lua`. For some reason, wcoss2 was using system python and not sure why that worked.
> > 
> > 
> > @aerorahul Could you let me know how to load python module in `module_gwsetup.wcoss2.lua` specifically?
> 
> @JongilHan66 A fix was merged in #3053. The change can be seen [here](https://github.com/NOAA-EMC/global-workflow/pull/3053/files#diff-4a96a6a951830b66c158250c0b7fca2be8240ed1b3e25059f9742cd419f91073).

@aerorahul Thanks!!
